chef/automate
# Adding new desktop node and run fields

### :nut_and_bolt: Description: What code changed, and why?

Adding needed fields from the ohai data to the infra node and infra runs. These added fields will be used with the new Desktop views. 

### :chains: Related Resources

https://github.com/chef/automate/issues/3320

### :athletic_shoe: How to Build and Test the Change
1. `build components/config-mgmt-service && build components/ingest-service && build components/automate-gateway && start_all_services`
1. Add a infra node with `generate_chef_run_example | jq '.entity_uuid = "832e3d1c-8e3e-11ea-bc55-0242ac130003" | .run_id = "9574da6c-8e3e-11ea-bc55-0242ac130003" | .id = "9574da6c-8e3e-11ea-bc55-0242ac130003"' | send_chef_data_raw`. 
1. Run this command `curl -f --insecure -H "api-token: $(get_api_token)" "https://localhost/api/v0/cfgmgmt/nodes/832e3d1c-8e3e-11ea-bc55-0242ac130003/runs/9574da6c-8e3e-11ea-bc55-0242ac130003" | jq | less`
1. Ensure these fields ip6address, timezone, domain, hostname, memory_total, macaddress, dmi_system_serial_number, dmi_system_manufacturer, virtualization_role, virtualization_system, kernel_version, kernel_release, and cloud_provider are present. (hint: they are all on the bottom)

### :white_check_mark: Checklist
